mlrun/feature_store/feature_set.py

```diff
@@ -812,6 +830,14 @@ def to_dataframe(
             if self.spec.timestamp_key and self.spec.timestamp_key not in entities:
                 columns = [self.spec.timestamp_key] + columns
             columns = entities + columns
+
+        if self.spec.passthrough:
+            if not self.spec.source:
+                raise mlrun.errors.MLRunNotFoundError(
+                    "passthrough feature set with no source bounded"
```

Comment:
```suggestion
                    f"passthrough feature set {self.metadata.name} with no source"
```
as per the other place this message is raised.